libs/deps/merch-card.js

```diff
@@ -797,6 +797,7 @@ merch-card[variant="mini-compare-chart"] [is="inline-price"] {
     display: inline-block;
     min-height: 30px;
     min-width: 1px;
+    word-break: break-all;
```

Comment:
how would this work with JP?

Reply:
![Screenshot 2024-06-21 at 10 15 48 AM](https://github.com/adobecom/milo/assets/1179549/129d5ad9-0c43-4134-9671-2dc35ced041c)
https://main--cc--adobecom.hlx.live/jp/creativecloud?milolibs=MWPW-151429--milo--adobecom#